pkg/authorization/authorizer/authorizer.go

```diff
@@ -3,30 +3,38 @@ package authorizer
 import (
 	"errors"
 
-	kerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
-
-	"github.com/openshift/origin/pkg/authorization/rulevalidation"
+	kauthorizer "k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+	authorizerrbac "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac"
 )
 
 type openshiftAuthorizer struct {
```

Comment:
I wasn't expecting us to have this anymore.  I will keep reading, but if this is on its way out let's have a comment.